demos/continuous_batching/README.md

```diff
@@ -161,20 +170,46 @@ curl http://localhost:8000/v3/chat/completions \
     }
   ],
   "created": 1716825108,
-  "model": "meta-llama/Llama-2-7b-chat-hf",
+  "model": "meta-llama/Meta-Llama-3-8B-Instruct",
   "object": "chat.completion"
 }
 ```
 
-## Streaming:
+A similar call can be made with a `completion` endpoint:
+```bash
+curl http://localhost:8000/v3/completions \
+  -H "Content-Type: application/json" \
+  -d '{
+    "model": "meta-llama/Meta-Llama-3-8B-Instruct",
+    "max_tokens":30,
+    "stream":false,
+    "prompt": "<|begin_of_text|><|start_header_id|>system<|end_header_id|>\n\nYou are assistant<|eot_id|><|start_header_id|>user<|end_header_id|>\n\nWhat is OpenVINO?<|eot_id|><|start_header_id|>assistant<|end_header_id|>"
+  }'| jq .
+```
+```json
+{
+  "choices": [
+    {
+      "finish_reason": "stop",
+      "index": 0,
+      "logprobs": null,
+      "text": "\n\nOpenVINO is an open-source software library for deep learning inference that is designed to optimize and run deep learning models on a variety"
+    }
+  ],
+  "created": 1716825108,
+  "model": "meta-llama/Meta-Llama-3-8B-Instruct",
+  "object": "text_completions"
+}
+```
+
+### Streaming:
 
-The endpoint `chat/completions` is compatible with OpenAI client so it can be easily used to generated code also in streaming mode:
+The endpoints `chat/completions` are compatible with OpenAI client so it can be easily used to generated code also in streaming mode:
```

Comment:
```suggestion
The endpoints `chat/completions` are compatible with OpenAI client so it can be easily used to generate code also in streaming mode:
```